.pipelines/templates/build-package-for-macosx.yml

```diff
@@ -20,11 +20,11 @@ stages:
   - template: mac-shared-lib-build.yml
     parameters:
       MacosArch: 'arm64'
-      AdditionalBuildFlags: "${{ parameters.AdditionalBuildFlags }} CMAKE_OSX_ARCHITECTURES=arm64"
+      AdditionalBuildFlags: "${{ parameters.AdditionalBuildFlags }} --cmake_extra_defines CMAKE_OSX_ARCHITECTURES=arm64"
```

Comment:
good change. relying on `--cmake_extra_defines` preceding this is a bit confusing.
FYI I added [`--one_cmake_extra_define`](https://github.com/microsoft/onnxruntime-extensions/blob/68b9d1dc47663a9017c55d136c804417c8efec7d/tools/build.py#L131-L137) which takes one argument for situations like this.